core/src/main/java/org/mule/runtime/core/internal/lifecycle/phases/MuleContextStopPhase.java

```diff
@@ -63,7 +68,19 @@ public MuleContextStopPhase(Class<?>[] ignorredObjects) {
         Stoppable.class
     });
 
-    setIgnoredObjectTypes(ignorredObjects);
+    if (parseBoolean(getProperty(REUSE_GLOBAL_ERROR_HANDLER_PROPERTY))) {
```

Comment:
What happens here with the ignoredObjects as parameters?

Reply:
fixed